Makefile

```diff
@@ -122,19 +138,28 @@ an existing lotus binary in your PATH. This may cause problems if you don't run
 
 .PHONY: build
 
-install: install-daemon install-miner install-worker
+install: install-daemon install-miner install-worker install-services install-completions
+
+install-daemon: lotus $(BINDIR)
+	install -o root -g root -m 755 -s ./lotus $(BINDIR)/lotus
+	install -C -o root -g root -m 755 ./scripts/lotus-init.sh $(BINDIR)/lotus-init.sh
```

Comment:
We shouldn't be specifying the user/group.

Reply:
The deb package should be run with fakeroot, right? Is the default user not correct?